go/vt/vtgate/planbuilder/collations_test.go

```diff
@@ -20,12 +20,11 @@ import (
 	"fmt"
 	"testing"
 
-	"vitess.io/vitess/go/test/vschemawrapper"
```

Comment:
What is this change doing here?

Reply:
*sigh*

My IDE will auto-cleanup imports when a file is open. I must have had this file open while I switched branches. I fix.